src/openms/include/OpenMS/ANALYSIS/XLMS/OPXLHelper.h

```diff
@@ -218,7 +218,7 @@ namespace OpenMS
       static void addPercolatorFeatureList(ProteinIdentification& prot_id);
 
       /**
-       * @brief adds the delta score as a MetaValue
+       * @brief adds the delta score as a MetaValue, assumes the hits under each PeptideIdentification are ordered by score rank
```

Comment:
can you add an OPENMS_PRECONDITION that checks if it is sorted?

Reply:
is there a quick way to check if PeptideHits are sorted by score?

Reply:
ah Ok, PeptideIdentification has a function to sort Hits by score. I guess I can just run that before calculating the delta score to make sure as well.